pkg/httpmock/stub.go

```diff
@@ -192,6 +192,14 @@ func RESTPayload(responseStatus int, responseBody string, cb func(payload map[st
 	}
 }
 
+// RESTResponse returns a static response, without any attempt to parse the
+// request body
+func RESTResponse(responseStatus int, responseBody string) Responder {
+	return func(req *http.Request) (*http.Response, error) {
+		return httpResponse(responseStatus, req, bytes.NewBufferString(responseBody)), nil
+	}
+}
+
```

Comment:
**Should this really be replaced with using `StatusJSONResponse`?**

https://github.com/cli/cli/blob/2bd3c2290331cd38282fc8a82d810dd8c8a57b14/pkg/httpmock/stub.go#L163-L171

The only significant difference is whether there are headers returned, which shouldn't be impactful to the test here.